Src/BootCamp.Chapter/CredentialsManager.cs

```diff
@@ -0,0 +1,168 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace BootCamp.Chapter
+{
+    internal class CredentialsManager
+    {
+        private const string CredentialsPath = @"..\..\..\Database\Credentials.txt";
+
+        public CredentialsManager()
+        {
+            if (!File.Exists(CredentialsPath))
+            {
+                File.WriteAllText(CredentialsPath, "");
```

Comment:
use this instead:
```
File.Create(filename).Dispose();
```
It does just what you need- create a new file and close it.

Reply:
As requested, I am using the File.Create(filename).Dispose() command in place of the code that I was using.  The effect seems to be the same when testing.